modules/framework/galasa-parent/dev.galasa.framework/src/main/java/dev/galasa/framework/internal/runner/TestRunnerDataProvider.java

```diff
@@ -7,6 +7,7 @@
 
 import java.util.Properties;
 
+import dev.galasa.framework.BundleManager;
```

Comment:
Bit odd adding a new import but no code which uses it ?

Reply:
The BundleManager class was moved, the import was just updated to reflect its new location.